string[] link_names
geometry_msgs/PoseStamped[] link_poses
